### PR TITLE
Update config

### PR DESCRIPTION
**Pull request by wgordon17**

### CODE REVIEW

**Comment by aweiteka** [comment]
[test]

File: openshift_tools/ircbot/openshift_sre/openshift_sre.py [hunk 1]
```diff
@@ -345,21 +377,21 @@ def mark_channel_for_announcements(bot, trigger):
 def unmark_channel_for_announcements(bot, trigger):
```
Comment (aweiteka) [inline openshift_tools/ircbot/openshift_sre/openshift_sre.py:377]: RE: pylint error, it's above the 30-character limit for method name.

**Review by aweiteka — APPROVED** [review]

File: openshift_tools/ircbot/openshift_sre/openshift_sre.py (hunk 1 above)
Comment (wgordon17) [reply openshift_tools/ircbot/openshift_sre/openshift_sre.py:377]: Ahh, thanks! Any reason to shorten it just to appease pylint?

Comment (aweiteka) [reply openshift_tools/ircbot/openshift_sre/openshift_sre.py:377]: > Any reason to shorten it just to appease pylint?

I guess it's fine as-is. There's a bit of noise here that needs some pylint config tweaks to quiet, IMO.

File: openshift_tools/ircbot/sopel.cfg.example [hunk 2]
```diff
@@ -9,6 +9,10 @@ admins = admin1, admin2
 channels = #mybot
 exclude = url, ipython
 extra = /absolute/path/to/openshift-tools/openshift_tools/trello, /absolute/path/to/openshift-tools/openshift_tools/openshift_sre
+auth_method = userserv
```
Comment (wgordon17) [inline openshift_tools/ircbot/sopel.cfg.example:12]: @aweiteka Thoughts on changing our requirements.txt to use my fork that allows this to work? https://github.com/wgordon17/sopel/tree/add-userserv-auth

**Pull request merged**